README.md

```diff
@@ -1,17 +1,242 @@
+
+``` cs
+namespace ReviewService.Abstractions;
+
+/// <summary>
+/// Extensions for <see cref="IReviewConditionsBuilder{TReviewSettings}"/>.
+/// </summary>
+public static partial class ReviewConditionsBuilderExtensions
+{
+    /// <summary>
+    /// The application onboarding must be completed.
+    /// </summary>
+    /// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+    /// <param name="builder">The builder.</param>
+    /// <returns><see cref="IReviewConditionsBuilder{TReviewSettings}"/>.</returns>
+    public static IReviewConditionsBuilder<TReviewSettings> ApplicationOnboardingCompleted<TReviewSettings>(this IReviewConditionsBuilder<TReviewSettings> builder)
+        where TReviewSettings : ReviewSettingsCustom
+    {
+        builder.Conditions.Add(new SynchronousReviewCondition<TReviewSettings>(
+            (reviewSettings, currentDateTime) => reviewSettings.HasCompletedOnboarding is true)
+        );
+        return builder;
+    }
+}
+```
+
+It's possible to customize the review conditions used by the service by using `ReviewConditionsBuilder` and passing it to the `ReviewService` constructor or by injecting it as a transient when using dependency injection.
+
+``` cs
+using ReviewService;
+
+var reviewConditionsBuilder = ReviewConditionsBuilder.Empty<ReviewSettingsCustom>()
+    .ApplicationOnboardingCompleted()
+    .MinimumPrimaryActionsCompleted(3)
+    .MinimumApplicationLaunchCount(3)
+    .MinimumTimeElapsedSinceApplicationFirstLaunch(TimeSpan.FromDays(5));
+```
+
+### Track Review Settings
+
+To track your custom review settings, you can create extensions for `IReviewService` and be sure to make them generic so they are usable with custom review settings.
+
+``` cs
+using ReviewService.Abstractions;
+
+/// <summary>
+/// Extensions of <see cref="IReviewService{TReviewSettings}"/>.
+/// </summary>
+public static class ReviewServiceExtensions
+{
+    /// <summary>
+    /// Tracks that the application onboarding has been completed.
+    /// </summary>
+    /// <typeparam name="TReviewSettings">The type of the object that we use for tracking.</typeparam>
+    /// <param name="reviewService"><see cref="IReviewService{TReviewSettings}"/>.</param>
+    /// <param name="ct">The cancellation token.</param>
+    /// <returns><see cref="Task"/>.</returns>
+    public static async Task TrackOnboardingCompleted<TReviewSettings>(this IReviewService<TReviewSettings> reviewService, CancellationToken ct)
+        where TReviewSettings : ReviewSettingsCustom
+    {
+        await reviewService.UpdateReviewSettings(ct, reviewSettings =>
+        {
+            return reviewSettings with { HasCompletedOnboarding = reviewSettings.HasCompletedOnboarding };
+        });
+    }
+}
+```
+
+## Testing
+
+This is what you need to know before testing and debugging this service.
```

Comment:
Mention that this may change and people should always refer to the Apple and Google documentation for the most up-to-date information.